### PR TITLE
Added info on all-in-one installation

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1312343

### CODE REVIEW

File: install_config/install/planning.adoc [hunk 1]
```diff
@@ -79,14 +79,21 @@ Limits] for the latest limits for objects in {product-title}.
...
+{product-title} cluster, based on your xref:sizing[sizing] needs.
 
 [NOTE]
 ====
 Moving from a single master cluster to multiple masters after installation is
 not supported.
 ====
 
+[[single-master-single-box]]
+=== Single Master and Nodes on One System
```
Comment (jianlinliu) [inline install_config/install/planning.adoc]: Should correct "Nodes" to "Node", right?

**Review by jianlinliu — COMMENTED** [review]
correct typo.

**Comment by mburke5678** [comment]
[rev_history]
|xref:../install_config/install/planning.adoc#install-config-install-planning[Installation and Configuration -> Planning]
|Added a new section on installing a xref:../install_config/install/planning.adoc#single-master-single-box[Single Master and Node on One System].
%

**Comment by mburke5678** [comment]
@openshift/team-design PTAL XS change

**Comment by bmcelvee** [comment]
LGTM 🌞

**Pull request merged**